### ISSUE TITLE
404 in Interface guidelines > Accessibility tools article

### ISSUE THREAD

**Issue by juliyvchirkov** [primer/design#145]
hi guys

first of all let me express my appreciations and thankfulness for the hight quality precious stuff you develop and share

eploring [primer.style](https://primer.style/), I've faced the broken link in the article [Interface guidelines > Accessibility tools](https://primer.style/design/accessibility/tools) (repo source: [content/accessibility/tools.mdx](https://github.com/primer/design/blob/master/content/accessibility/tools.mdx])), which points to a missing resource at [figma.com](https://www.figma.com/)

the broken a href encloses the title of the first paragraph of the article (`Accessibility sticker sheet`), the destination [figma.com/file/o3Vhxw4SmBsUjeaVq3nF4y/Accessibility-Notation?node-id=0%3A1](https://www.figma.com/file/o3Vhxw4SmBsUjeaVq3nF4y/Accessibility-Notation?node-id=0%3A1) constantly reports `404` despite of visitor's status (if one logged on [figma.com](https://www.figma.com/) or not) and url escaping (both `?node-id=0%3A1` and `node-id=0:1` result in `404`), so I can assume the issue on [figma.com](https://www.figma.com/) side is  permanent and requires to be resolved by hands

[inspect the broken element](https://primer.style/design/accessibility/tools#accessibility-sticker-sheet)

<img src=https://i.imgur.com/II5Lf1v.png width="75%">

**Comment by yaili** [comment]
Hi, @juliyvchirkov 

Thanks for submitting this. I need to double check with the team, but I'm not sure this file is supposed to be public, in which case the text should be very clear about (so there's still an issue, but the fix would be different).

**Comment by yaili** [comment]
Update: We'll transfer this to the figma repo and update the file to be open source. Thanks for the report, @juliyvchirkov
